src/frontend/constants.ts

```diff
@@ -1,5 +1,6 @@
 export const URI_PREFIX = "mapeo://";
 export const ERROR_STORE_KEY = "@MapeoError";
+export const NAV_STORE_KEY = "@MapeoNav";
```

Comment:
As long as we're persisting navigation state, it's probably safer to preserve the version number (8) here (thinking about nav changes that are added later on that aren't related to experiments or feature flags)